app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <import type="android.text.InputType"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.PinPasswordViewModel" />
+  </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/pinPasswordBackground">
+    <TextView
+      android:id="@+id/hello_text"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:lines="2"
+      android:gravity="center_horizontal"
+      android:text="@{viewModel.profile.isAdmin ? `Hi, ` + viewModel.profile.name + `!\nPlease enter your Admin PIN` : `Hi, ` + viewModel.profile.name + `!\nPlease enter your PIN`}"
```

Comment:
This should be defined in strings file, check this string for exampled: `topic_download_text`

Reply:
Updated. Thanks for including an example!